src/ui/ui_render/ui_render_mixin.js

```diff
@@ -144,19 +165,21 @@ export function uiRenderMixin(kbnServer, server, config) {
 
   async function renderApp({ app, h, includeUserProvidedConfig = true, injectedVarsOverrides = {} }) {
     const request = h.request;
-    const translations = await server.getUiTranslations();
     const basePath = request.getBasePath();
 
     return h.view('ui_app', {
       uiPublicUrl: `${basePath}/ui`,
       bootstrapScriptUrl: `${basePath}/bundles/app/${app.getId()}/bootstrap.js`,
       i18n: (id, options) => i18n.translate(id, options),
-      locale: i18n.getLocale(),
 
       injectedMetadata: {
         version: kbnServer.version,
         buildNumber: config.get('pkg.buildNum'),
         basePath,
+        i18n: {
+          locale: i18n.getLocale(),
```

Comment:
__note:__ `locale` isn't used right now, but I thought it'd make sense to include it regardless, but let me know if you think we should add only when we need it.